components/kyma-environment-broker/internal/process/provisioning/initialisation.go

```diff
@@ -114,28 +109,22 @@ func (s *InitialisationStep) Run(operation internal.ProvisioningOperation, log l
 }
 
 func (s *InitialisationStep) initializeRuntimeInputRequest(operation internal.ProvisioningOperation, log logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
-	pp, err := operation.GetProvisioningParameters()
-	if err != nil {
-		log.Errorf("cannot fetch provisioning parameters from operation: %s", err)
-		return s.operationManager.OperationFailed(operation, "invalid operation provisioning parameters")
-	}
-
-	err = s.configureKymaVersion(&operation, &pp)
+	err := s.configureKymaVersion(&operation, &operation.ProvisioningParameters)
```

Comment:
Do you really need to pass the Provisioning Parameters explicitly? It is now a part of the operation so it can be read from there inside underlying methods.

Reply:
changed the underlying methods to use operation, I've found similar issue in `create_runtime.go` and I also adjusted that